src/util/borrowUtils.ts

```diff
@@ -26,3 +26,22 @@ export const useTotalFiatAmount = (wallet: EdgeCurrencyWallet, borrowArray: Borr
     }, '0')
   }, [allTokens, borrowArray, currencyInfo, exchangeRates, isoFiatCurrencyCode])
 }
+
+export const getWalletPickerExcludeWalletIds = (
+  wallets: { [wallet: string]: EdgeCurrencyWallet },
+  actionOpType: string,
+  borrowEngineWallet: EdgeCurrencyWallet
+) => {
+  return Object.keys(wallets).filter(walletId => {
+    switch (actionOpType) {
+      case 'loan-deposit':
+        return walletId !== borrowEngineWallet.id && wallets[walletId].currencyInfo.pluginId !== 'bitcoin'
+      case 'loan-borrow':
+      case 'loan-repay':
+      case 'loan-withdraw':
+        return walletId !== borrowEngineWallet.id
+      default:
+        return false
```

Comment:
Remove this case once you've exhausted the static type.

Reply:
Wait I take back what I said earlier. This default is still a good safegaurd for runtime in the case the type is extended. If the type is extended, there will be no errors.

Reply:
If the type is extended, the developer should consider the implications to this function when the type is extended. Why are we assuming that the correct behavior for some unknown future change to this type is for this function to return false?

Reply:
Ah missed this change

Reply:
This function restricts selectable wallets. If it was not properly updated, allow selection of any wallet so testing is not blocked